### ISSUE TITLE
Pseudohyperkalemia 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8155]
**Preferred term label:**
Pseudohyperkalemia 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pseudohyperkalemia describes a falsely elevated potassium level upon measurement of the serum electrolyte concentration due to disruption of cells during the collection or processing of the sample.

Pseudohyperkalemia is an elevation of potassium in the serum that results during collection, storage, or handling of a blood sample, and is not representative of the potassium level in vivo. Pseudohyperkalemia can result from increased tourniquet tension, increased pressure from a patient’s fist being clenched, or mechanical lysis of hematocytes during pneumatic tube delivery of samples to the laboratory or the centrifugation process
PMID:32953312

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
185020  - PMID: 16227998  no frequency

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
